assignment4/assignment4.js

```diff
@@ -0,0 +1,36 @@
+var fs = require('fs');
+var data1, data2;
+fs.readFile('file1.txt', function (err, data) {
```

Comment:
`err` and `data` should be descriptive. `err` should tell what error it is, like `errorOnFile1Read` and `data` should do similar, like `dataFromFile1`.